### PR TITLE
add Decentralized Docker Hub.md

### PR DESCRIPTION
**Pull request by viraja1**
add Decentralized Docker Hub.md

### CODE REVIEW

File: participants/Decentralized Docker Hub.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Decentralized Docker Hub
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name: Decentralized Docker Hub
+- Filecoin Slack handles: viraj
```
Comment (pooja) [inline participants/Decentralized Docker Hub.md:6]: Hey @viraja1! Excited to see your application come through.

File: participants/Decentralized Docker Hub.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+We are pushing docker images and not a curated dataset
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+Decentralized Docker Hub allows users to push any Docker image to IPFS and Filecoin
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+1T - 10T
```
Comment (pooja) [inline participants/Decentralized Docker Hub.md:32]: Could you say a little bit more about how you're planning on sourcing this data and storing it?

File: participants/Decentralized Docker Hub.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## (7) How are you structuring the data?
+
+Structured .tar file of the docker image
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+Structured .tar file of the docker image is being pushed. It also has support for ENS domain names. You just need to get the IPFS hash of the docker image and set it as the content hash of your ENS domain. You can then pull the docker image using ENS domain name or CID.
```
Comment (pooja) [inline participants/Decentralized Docker Hub.md:40]: What's the average size of these files? Will you have to split them into smaller chunks before storing to Filecoin? Current recommendation is to make deals for data chunks <= 4GB in size.

File: participants/Decentralized Docker Hub.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Using Decentralized Docker Hub, you can easily push and pull docker images from IPFS and filecoin using command line. It is powered by Powergate. It also has support for ENS domain names.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+We are pushing docker images and not a curated dataset
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+Decentralized Docker Hub allows users to push any Docker image to IPFS and Filecoin
```
Comment (dineshshenoy) [inline participants/Decentralized Docker Hub.md:28]: Please add this datasource to the curated datasets via a separate PR.

**Comment by dineshshenoy** [comment]
Congratulations. Your application has been approved and merged as an official participant for the Slingshot competition!
The following resources may be helpful as you begin:

[Competition Home](https://slingshot.filecoin.io/)
[Competition Rules](https://slingshot.filecoin.io/rules)
[FAQs](https://slingshot.filecoin.io/faq)
[Events calendar](https://slingshot.filecoin.io/events/) (Office Hours/Master Classes)
[Filecoin Docs](https://docs.filecoin.io) 
[HackFS videos](https://hack.ethglobal.co/showcase/myel-receOUiPSd63148qZ)
[Textile Powergate](https://docs.textile.io/powergate/)

Feel free to reach out in the slack community channels:
#fil-testnet-announce – For major announcements about the testnet and major software releases
#slingshot-announcements – For major announcements about the Slingshot program
#slingshot – For debugging help and community conversations throughout the course of the competition

**Pull request merged**